### ISSUE TITLE
General Enquiries about windows-history and applications like ditto

### ISSUE THREAD

**Issue by iamjonny** [sabrogden/Ditto#614]
Hi! Stellar work 👍🏻I just stumbled upon this whilst investigating issue noticed where clipboard doesn't perist after reboot (perhaps user logoff)  I'm not yet interested in ms-settings-sync using Microsoft Account ‼️

I'm wondering how you even made Ditto, if you had to rewrite following changes in MS architecture (or maybe the changes to the clipboard are not that big, even since XP...)

I've used nirsofts before; are these all using same approach? ("clipboardic" , you may have encountered)

Ultimately I wanted to _get_ the content, and understand where it is. I was about to procmon, maybe in safe mode to reduce _all that noise_ (noob on needle-hunt in giant haystacks)  but thought I'd ask an expert also.

TlDr is windows just using a SQLITE db, and can this be inferred, if not documented?

THANKS!
